geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/Stat.scala

```diff
@@ -79,7 +82,14 @@ trait Stat {
    *
    * @return stat as a json string
    */
-  def toJson: String
+  def toJson: String = Stat.JSON.toJson(toJsonObject)
+
+  /**
+    * Returns a representation of the stat to be serialized
+    *
+    * @return stat as a json serializable object
+    */
+  def toJsonObject: Any
```

Comment:
This might be a bigger question, but is there a 'closer' type bound we can use here?